tests/api_response_test.py

```diff
@@ -1,4 +1,3 @@
-# Copyright 2016-2017 Red Hat, Inc.
```

Comment:
Looks like a change that should be in the commit adding the spdx headers. It would be nice to move it to the right commit if possible.